pkg/proxy/healthcheck/proxier_health.go

```diff
@@ -92,6 +94,7 @@ func (hs *proxierHealthServer) QueuedUpdate() {
 func (hs *proxierHealthServer) Run() error {
 	serveMux := http.NewServeMux()
 	serveMux.Handle("/healthz", healthzHandler{hs: hs})
+	serveMux.Handle("/debug/flags/v", routes.StringFlagPutHandler(logs.GlogSetter))
```

Comment:
kube-proxy has a metrics and a healthcheck server

https://github.com/kubernetes/kubernetes/blob/661eae7c7c8226a3f7bed3a24acf1c22861477b7/cmd/kube-proxy/app/server.go#L666-L670

IMHO the health server doesn't seems the right place , should we open an issue first to discuss this?

Reply:
@aojea Thanks for review.  

The set-log-level handler needs to be installed to metrics server and controlled by 'profiling' flag. 

I just found an issue related to this flag: #98302. I'll close this PR for now, and request another PR when issue #98302 got fixed.